src/components/ToDos/ToDoCard/index.js

```diff
@@ -0,0 +1,29 @@
+import {useState} from 'react';
+
+import NewToDoButton from '../NewToDoButton';
+import ToDo from '../ToDo';
+import ToDoCardSection from '../ToDoCardSection';
+import ToDoCardTitle from '../ToDoCardTitle';
+
+import {ToDoCardContainer} from './styled';
+
+export default function ToDoCard() {
+	const [items, setItems] = useState([]);
+
+	function addNewToDos(items) {
+		setItems(prevItem => {
+			return [items, ...prevItem];
+		});
+	}
```

Comment:
The naming of this function does not make sense to me. As far as I can see, `items` is just one todo which is added, right? And this function basically adds one new todo to your state? If so, then I would recommend the following rewording:
```suggestion
	function addNewToDo(newTodo) {
		setItems(prevItems => {
			return [newTodo, ...prevItems];
		});
	}
```